### ISSUE TITLE
T3 - silhouette score

### ISSUE THREAD

**Issue by mathiasvaldebenito** [IIC2433/Syllabus-2020-2#101]
Viendo la documentación del método en sklearn vi que este recibe tres (o más parámetros) entre ellos: X, labels y metric. Mi duda es qué es este parámetro labels, leyendo solo la documentación no me logró quedó claro qué es.

**Comment by AstridESMJ** [comment]
¡Hola!

Si revisas la documentación de sklearn:
sklearn.metrics.silhouette_score(X, labels, *, metric='euclidean', sample_size=None, random_state=None, **kwds)
ese es el método y en particular para el parámetro labels : Predicted labels for each sample.
Es decir debes darle el label de la predicción hecha por el algoritmo a cada dato.
Es una medida de calidad del clustering realizado, qué tan "buenos" son los clusters generados.

Saludos,

Astrid

**Comment by mathiasvaldebenito** [comment]
y la predicción es un identificador del cluster dónde cayó el dato?

**Comment by Hernan4444** [comment]
Hola, recomiendo revisar la AC07 del curso (el enlace se mandó por aviso de canas), en esa se presenta dicha métrica, cuando un valor de ella es bueno y enlaces que serán de utilidad. Entre esos, uno que muestra cómo usar la función que preguntas.

Saludos :v:
